### ISSUE TITLE
Lesson maintainers?

### ISSUE THREAD

**Issue by ewallace** [carpentries-incubator/good-enough-practices#94]
Checking about the lesson maintainers.

The lesson still has @tzielins and @aromanowski listed but there's been limited involvement for a while - would either of you be interested in staying as maintainers or should we remove you?

Anyone else interested in contributing as a maintainer, reply in this thread and ask any questions that you need to?

**Comment by tzielins** [comment]
Yes @ewallace you should remove TZ and AR we only contributed little a the very beginning.

**Comment by alee** [comment]
I should be able to step in and help out as a maintainer but would definitely benefit from additional maintainers as I'm already maintaining the python-novice-gapminder lesson for the software carpentries and spread quite thin across too many projects 😅 

I've been in the good practices for open computational science space for some time now with a focus on computational modeling of social-ecological systems and have been involved with FAIR4RS, [comses.net](https://comses.net), [the Open Modeling Foundation](https://openmodelingfoundation.org), the FORCE11 Software Citation Implementation Working Group, etc. The material that's here is very much in line with other educational materials we'd like to develop over at [comses.net](https://comses.net/education) and it'd be great to see how to coordinate our efforts there.

**Comment by ewallace** [comment]
Hello @alee, I'm sorry we missed this back in March! Currently the de facto maintainers are myself (@ewallace), and along with @ameynert and @emma-wilson we are working through peer review comments to have this lesson accepted on carpentries lab and published at the journal of open science education.

The review thread is here: https://github.com/carpentries-lab/reviews/issues/24#issuecomment-1523302893

I intend to stay as a maintainer for at least the next year.

We would really appreciate you coming on board as a maintainer, and coordinating efforst for better online materials. Are you still interested?

**Comment by emma-wilson** [comment]
Hi @ewallace, I'm happy to continue as a maintainer 😄

**Comment by alee** [comment]
Apologies for the late reply, was out of the country for a few weeks and just getting back into the swing of things. I'd be happy to serve as a maintainer! And will take a closer look at the review thread you linked when I get some free time in the next week or two.

I taught parts of this course at a computational modeling spring school here at ASU and may have a chance here in a few weeks for another accelerated runthrough for another group of researchers who work in public health / computational modeling.

**Comment by ewallace** [comment]
Fixed by #118
